src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/processes-procedures/tech-writer-workflow.mdx

```diff
@@ -0,0 +1,134 @@
+
+1. First, ensure your **Current Branch** in Github Desktop is set to the correct branch, not **Develop**.
+2. Navigate to the doc you want to edit in Finder. If I wanted to edit a Python agent doc, I would navigate to: `~/Documents/Github/docs-website/src/content/docs/agents/python-agent/hosting-services/python-agent-stackato.mdx`
+3. Edit the doc in your text-editor of choice. You should write docs in markdown language. Reference the style guide for help with formatting markdown
+4. Save the file with your edits, then follow the same process for any other docs you wish to edit.
+
+## Committing your changes
+
+Once your edits are done, you can commit them. This stages your changes, which you will later push upstream.
+
+1. Navigate to Github Desktop. The left column should have a record of all the edits you have made to docs. 
+2. In the bottom left corner, name your commit and add a good description of your edits. It should be descriptive enough to ensure that someone can understand all the changes made by simply scanning this description.
+3. Press **Commit to yourbranchname**
+
+## Publish your commits
+
+Once you have committed your changes, you are almost ready to open your Pull Request.
+
+First, you need to ensure your branch is pushed upstream.
+
+1. On Github desktop, click the blue **Publish Branch** button if available.
+2. If you do not see the Publish Branch, click the blue **Push Origin** button. 
+
+This will push all your commits upstream and make them available to everyone else through the Github repository.
+
+## Open your pull request
+
+Now that your commits are available to everyone, you need to notify people that your changes are ready to be merged into the main branch (**develop**).
+
+To do this, you open a pull request:
+
+1. On Github Desktop, click the blue **Create Pull Request button**.
+2. This will open Github in your browser, and prompt you to fill in your pull request. If you scroll down, you can review all your commits to ensure they reflect all your changes.
+3. Just like your commit description, your pull request description should be detailed and give the full context of your changes. 
+    - Feel free to add any additional context here (issue or Jira number, SMEs, etc.)
+4. Once you are satisfied with your pull request, click the green **Create pull request button**. 
+5. You can either publish the changes directly by approving the pull request yourself, or you can request for another Tech Writer to peer edit it. 
+    - To request a review from another Tech Writer: in GitHub open the PR, navigate to the conversation **Conversation**, and then select or type in a reviewer name in the **Reviewer** section. 
+6. Once the pull request has been approved by another tech writer or you are confident the changes are ready to be published, click the green **Merge pull request** button. This will merge your branch and commits into the repository and should now be reflected in the docs site. 
+
```

Comment:
"should now be reflected" sounds like it will be right away. Something like, "and will begin the build process, which takes about ten minutes."